index.js

```diff
@@ -1 +1,220 @@
 console.log("Hello!!!");
+
+//Unique values
+// Check if word in anagrams arr is anagram
+// If you found anagram store this word in set or map
+// Values in set or map should not repeat anagram of this word
+// Also should not be present in set or map
+// Try to optimize script by using check has
+// Return collection set or map
+
+// Example: "actor" and "carot" are anagrams so store only "actor"
+
+const anagrams = [
+  "actor",
+  "carot",
+  "listen",
+  "enlist",
+  "debit",
+  "bidet",
+  "tear",
+  "rate",
+  "night",
+  "thing",
+  "lives",
+  "veils",
+  "stressed",
+  "desserts",
+  "dormitory",
+  "dirty room",
+  "rescue",
+  "secure",
+  "a gentleman",
+  "elegant man",
+  "listen",
+  "heart",
+  "angel",
+  "leader",
+  "silent",
+];
+
+const unique = (arr) => {
+  let x;
+  let y;
+  let z;
+  let litters = new Set();
+  let uniqueAnagrams = [];
```

Comment:
litters and uniqueAnagrams should be conts
litters maybe correct name is letters?